specification.markdown

```diff
@@ -313,35 +313,47 @@ The `name` provides the official human-readable name of the `EBML Element`. The
 
-A boolean to express if an `EBML Element` MUST occur at its defined level or may occur within any `Parent Element`.
+The `MinOccurence` represent the minimum number of occurrences of this `EBML Element` within its `Parent Element`. Each instance of the `Parent Element` MUST contain at least this many instances of this `EBML Element`. If the `EBML Element` has an empty `EBMLParentPath` then `MinOccurence` refers to constaints on the occurrence of the `EBML Element` within the `EBML Document`. The semantic meaning of `MinOccurence` within an `EBML Schema` is considered analogous to the meaning of `minOccurs` within an `XML Schema`. `EBML Elements` with `MinOccurence` set to "1" that also have a `default` value (see [default](#default)) declared are not REQUIRED to be stored but are REQUIRED to be interpreted, see [Note on the Use of default attributes to define Mandatory EBML Elements](#note-on-the-use-of-default-attributes-to-define-mandatory-ebml-elements). An `EBML Element` defined with a `MinOccurence` value greater than zero is called a `Mandatory EBML Element`.
 
-The `global` attribute is OPTIONAL. If the `global` attribute is not present then that `EBML Element` is to be considered not global.
+If `MinOccurence` is not present then that `EBML Element` is considered to have a `MinOccurence` value of 0.
 
-#### id
+The `MaxOccurence` represent the maximum number of occurrences of this `EBML Element` within its `Parent Element`. Each instance of the `Parent Element` MUST contain at most this many instances of this `EBML Element`. If the `EBML Element` has an empty `EBMLParentPath` then `MaxOccurence` refers to constaints on the occurrence of the `EBML Element` within the `EBML Document`. The term `unbounded` indicates there is no maximum number of occurrences. The semantic meaning of `MaxOccurence` within an `EBML Schema` is considered analogous to the meaning of `maxOccurs` within an `XML Schema`.
 
-The `Element ID` encoded as a `Variable Size Integer` expressed in hexadecimal notation prefixed by a `0x` that is read and stored in big-endian order. To reduce the risk of false positives while parsing `EBML Streams`, the `Element IDs` of the `Root Element` and `Top-Level Elements` SHOULD be at least 4 octets in length. `Element IDs` defined for use at Level 0 or Level 1 MAY use shorter octet lengths to facilitate padding and optimize edits to `EBML Documents`; for instance, the `Void Element` uses an `Element ID` with a one octet length to allow its usage in more writing and editing scenarios.
+If `MaxOccurence` is not present then that `EBML Element` is considered to have a `MaxOccurence` value of 1.
 
-The `id` attribute is REQUIRED.
-
-#### minOccurs
+The `"()+"` group of the `EBMLElementPath` specify that the `EBML Element` can occur within itself recursively (see [Identically Recurring Element](#identically-recurring-elements)).
```

Comment:
This seems like a different meaning of `Identically Recurring Element`. I thought an `Identically Recurring Element` would be like copies of the `Tracks` element occurring every once in a while within a Matroska stream, without recursive placement.